test/unit/vl-map-layer-util.js

```diff
@@ -0,0 +1,105 @@
+const LAYER_SELECTOR = '[data-vl-is-layer]';
```

Comment:
Dit ruikt naar testen op elke extend van een abstract. Is het dan niet beter om de abstract alleen te testen?

Reply:
Dan weet je niet 100% zeker of de extend gelukt is en niet overridden wordt op niveau van de subklasse. Daarmee dat ik het zo maar gedaan had.

Reply:
Lijkt me goed, maar ik ga eens kijken of ik de code iets anders kan organiseren zodat er minder code en geen util meer nodig is.

Reply:
Refactor uitgevoerd waardoor utils niet meer nodig zijn en de testen dichter bij de bron zitten maar toch op al de extends uitgevoerd worden.